src/frontend/pages/_document.tsx

```diff
@@ -22,6 +22,8 @@ export default class MyDocument extends Document<{ envString: string }> {
       const baggage = propagation.getBaggage(context.active());
       const isSyntheticRequest = baggage?.getEntry('synthetic_request')?.value === 'true';
 
+      console.log(``)
```

Comment:
Is this required?

Reply:
Ugh I need to add some eslint to the build. I'll remove.